Projects/App/Sources/Present/Detail/ViewController/DetailPostViewController.swift

```diff
@@ -238,11 +187,11 @@ final class DetailPostViewController: BaseVC<DetailPostViewModel>, CommentDataPr
             vc: self)
     }
     
-    private func presentFeaturePreparationAlert() {
+    private func presentFailShareAlert() {
```

Comment:
```suggestion
    private func presentFailedShareAlert() {
```

실패되었을 때라는 의미를 포함하도록 이렇게 변경하는게 어떨까요?

Reply:
좋은 생각입니다.

Reply:
수정했습니다 [775b764](https://github.com/SLTDV/Choice-iOS/pull/162/commits/775b764cfc434480a357ddfb7e1d8749833b9b4d)